datasets/gov/lsd/it_naudojimas_namu_ukiuose.csv

```diff
@@ -0,0 +1,2514 @@
+,,,,,,enum,,,"""1""",,,,Taip,
+,,,,,,,,,"""0""",,,,Ne,
+,,,,,,,,,"""9""",,,,Netaikoma,
+,,,,,sbmob,string,,,,4,open,,"Ar per pastaruosius 12 mėnesių abejonės dėl saugumo, naudojantis internetu asmeniniais tikslais, sulaikė Jus nuo toliau išvardytų veiksmų: Naudojimosi internetu per mobilųjį įrenginį, (pvz., per nešiojamąjį kompiuterį) belaidžiu ryšiu ne namuose",
+,,,,,,enum,,,"""1""",,,,Taip,
+,,,,,,,,,"""0""",,,,Ne,
+,,,,,,,,,"""9""",,,,Netaikoma,
+,,,,,sav1,string,,,,4,open,,"Ar Jūs darote atsargines failų (dokumentų, nuotraukų ir kt.) kopijas iš kompiuterio į bet kokias išorines laikmenas, pvz., CD, DVD, išorinį kietąjį diską, USB laikmeną ar interneto duomenų saugykla? ",
+,,,,,,enum,,,"""1""",,,,Taip,
+,,,,,,,,,"""0""",,,,Ne,
+,,,,,,,,,"""9""",,,,Netaikoma,
+,,,,,cook,string,,,,4,open,,"Ar Jūs žinote, kad slapukai (angl. cookies) gali būti panaudoti turint tikslą sekti interneto naudotojo veiklą internete, sudaryti kiekvieno vartotojo profilį ir pateikti jiems pritaikytą reklamą?",
+,,,,,,enum,,,"""1""",,,,Taip,
+,,,,,,,,,"""0""",,,,Ne,
+,,,,,,,,,"""9""",,,,Netaikoma,
+,,,,,pcook,string,,,,4,open,,"Ar Jūs kada nors keitėte interneto naršyklės nuostatas, kad apsisaugotume nuo slapukų ar apribotumėte jų kiekį?",
+,,,,,,enum,,,"""1""",,,,Taip,
+,,,,,,,,,"""0""",,,,Ne,
+,,,,,,,,,"""9""",,,,Netaikoma,
+,,,,,rf_age,string,,,,4,open,,Rizikos faktorius - amžius,
+,,,,,,enum,,,"""1""",,,,Yra,
+,,,,,,,,,"""0""",,,,Nėra,
+,,,,,dsk_i,string,,,,4,open,,Informaciniai įgūdžiai,
+,,,,,,enum,,,"""0""",,,,Nėra,
+,,,,,,,,,"""1""",,,,Baziniai,
+,,,,,,,,,"""2""",,,,Aukštesni nei baziniai,
+,,,,,,,,,"""9""",,,,Netaikoma ,
+,,,,,dsk_c,string,,,,4,open,,Komunikaciniai įgūdžiai,
+,,,,,,enum,,,"""0""",,,,Nėra,
+,,,,,,,,,"""1""",,,,Baziniai,
+,,,,,,,,,"""2""",,,,Aukštesni nei baziniai,
+,,,,,,,,,"""9""",,,,Netaikoma ,
+,,,,,dsk_ps,string,,,,4,open,,Problemų sprendimo įgūdžiai,
+,,,,,,enum,,,"""0""",,,,Nėra,
+,,,,,,,,,"""1""",,,,Baziniai,
+,,,,,,,,,"""2""",,,,Aukštesni nei baziniai,
+,,,,,,,,,"""9""",,,,Netaikoma ,
+,,,,,dsk_s,string,,,,4,open,,Darbo su programų paketais įgūdžiai,
+,,,,,,enum,,,"""0""",,,,Nėra,
+,,,,,,,,,"""1""",,,,Baziniai,
+,,,,,,,,,"""2""",,,,Aukštesni nei baziniai,
+,,,,,,,,,"""9""",,,,Netaikoma ,
+,,,,,dsk,string,,,,4,open,,Bendri įgūdžiai,
+,,,,,,enum,,,"""0""",,,,Nėra,
+,,,,,,,,,"""1""",,,,Žemesni nei baziniai,
+,,,,,,,,,"""2""",,,,Baziniai,
+,,,,,,,,,"""3""",,,,Aukštesni nei baziniai,
+,,,,,rf_edu1,string,,,,4,open,,Rizikos faktorius - išsilavinimas,
+,,,,,,enum,,,"""1""",,,,Yra,
+,,,,,,,,,"""0""",,,,Nėra,
+,,,,,rf_empst,string,,,,4,open,,Rizikos faktorius - užimtumo statusas,
+,,,,,,enum,,,"""1""",,,,Yra,
+,,,,,,,,,"""0""",,,,Nėra,
+,,,,,stratum_id,string,,,,4,open,,,
+,,,,,psu,string,,,,4,open,,,
+,,,,,,,,,,,,,,
+,,,,ItAsmenu,,,,,,,3,,Asmenų informacinių technologijų naudojimas,Lietuvos statistikos departamento atlikti tyrimai

```

Comment:
Nėra pirminio rakto (jeigu toks turi būti). Taip pat, kuo šis modelis skiriasi nuo ItNaudAsmuo modelio, nes title yra toks pat?

Reply:
Čia yra tik du modeliai, ištaisyta.